src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -317,18 +318,16 @@ export class MeetingManager implements AudioVideoObserver {
       configuration,
       logger,
       deviceController,
-      this.eventReporter
+      this.eventController
     );
+    if(!this._eventController)
+    {
+      this._eventController = this.meetingSession.eventController;
+      this.eventController.addObserver(this.eventDidReceiveRef);
```

Comment:
This syntax is a bit confusing and I know that `this.eventController` is a getter (might affect understanding if not known already 😅 ), but would prefer to use the `this._eventController.addObserver`.

Reply:
Sure, updated